src/Monolog/Handler/LogglyHandler.php

```diff
@@ -15,12 +15,12 @@
 use Monolog\Formatter\LogglyFormatter;
 
 /**
- * Sends errors to Loggly.
- *
- * @author Przemek Sobstel <przemek@sobstel.org>
- * @author Adam Pancutt <adam@pancutt.com>
- * @author Gregory Barchard <gregory@barchard.net>
- */
+* Sends errors to Loggly.
+*
+* @author Przemek Sobstel <przemek@sobstel.org>
+* @author Adam Pancutt <adam@pancutt.com>
+* @author Gregory Barchard <gregory@barchard.net>
+*/
```

Comment:
@kamilsk  this is not the change, I will fix it. the actual fix is to add retry logic in case of timeouts and other recoverable server errors. Please see below for the changes,